cms/envs/common.py

```diff
@@ -459,17 +459,17 @@
         ],
         'output_filename': 'css/cms-style-vendor-tinymce-skin.css',
     },
-    'style-app': {
+    'style-main': {
         'source_filenames': [
-            'sass/style-app.css',
+            'sass/studio-main.css',
         ],
-        'output_filename': 'css/cms-style-app.css',
+        'output_filename': 'css/studio-main.css',
     },
-    'style-app-rtl': {
+    'style-main-rtl': {
         'source_filenames': [
-            'sass/style-app-rtl.css',
+            'sass/studio-main-rtl.css',
         ],
-        'output_filename': 'css/cms-style-app-rtl.css',
+        'output_filename': 'css/studio-main-rtl.css',
```

Comment:
@feanil, will this have any cache/production repercussions when this gets merged and a release is cut?